sds_data_manager/constructs/ialirt_ingest_lambda_construct.py

```diff
@@ -133,10 +135,57 @@ def create_algorithm_dynamodb_table(self) -> aws_dynamodb.Table:
         )
         return self.algorithm_data_table
 
+    def create_table(self) -> aws_dynamodb.Table:
+        """Create and return the algorithm data product table."""
+        removal_policy = (
+            RemovalPolicy.RETAIN
+            if self.account_name == "prod"
+            else RemovalPolicy.DESTROY
+        )
+        point_in_time_recovery = True if self.account_name == "prod" else False
+
+        self.data_table = ddb.Table(
+            self,
+            "IalirtDataTable",
+            table_name="ialirt-data-table",
+            # RemovalPolicy.RETAIN to keep the table after stack deletion.
+            removal_policy=removal_policy,
+            # Restore data to any point in time within the last 35 days.
+            point_in_time_recovery=point_in_time_recovery,
+            # Partition key (PK) = instrument.
+            partition_key=ddb.Attribute(
+                name="instrument",
+                type=ddb.AttributeType.STRING,
+            ),
+            # Sort key (SK) = met_in_utc.
+            sort_key=ddb.Attribute(
+                name="met_in_utc",
```

Comment:
If this is going to be storing the measurements I think we should avoid MET because sometimes it came down in multiple packets.
```suggestion
                name="time_utc",
```